js/functional/palindrome.js

```diff
@@ -16,7 +28,16 @@ function isPalindromeUsingReverse(value) {
  * @param {string} value
  */
 function isPalindromeUsingLoop(value) {
-  // write your code here
+  value = parseString(value);
+
+  const cleanedValue = cleanString(value);
+  const length = cleanedValue.length;
+  for (let index = 0; index < length / 2; index++) {
```

Comment:
Put an extra line break at this "for" statement to make it easier to read by the other.